src/main/java/mate/academy/spring/controller/InjectController.java

```diff
@@ -0,0 +1,44 @@
+package mate.academy.spring.controller;
+
+import java.util.Set;
+import mate.academy.spring.model.Role;
+import mate.academy.spring.model.User;
+import mate.academy.spring.service.RoleService;
+import mate.academy.spring.service.UserService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/inject")
+public class InjectController {
```

Comment:
Inject data using DataInitializer and PostConstruct
![image](https://user-images.githubusercontent.com/32339836/235959127-ed64ac31-0a0f-4d24-b853-b16e800b3b51.png)